packages/embed-react/src/components/inline-style.spec.tsx

```diff
@@ -36,6 +36,7 @@ describe('<InlineStyle />', () => {
       </>
     )
 
+    // eslint-disable-next-line prefer-destructuring
     const styleElement = screen.getByTestId('style').getElementsByTagName('style')[0]
```

Comment:
How about destructuring instead?
```suggestion
    const [ styleElement ] = screen.getByTestId('style').getElementsByTagName('style')
```

Reply:
Tried and got `Type 'HTMLCollectionOf<HTMLStyleElement>' is not an array type.` from Typescript 😞 Not sure why, because I don't see that complaint in the IDE 🤔